client/signup/config/steps.js

```diff
@@ -71,6 +71,13 @@ module.exports = {
 		providesDependencies: [ 'cartItem' ]
 	},
 
+	'select-plan': {
```

Comment:
Multiple components in signup expect the key for the step and the `stepName` property of the step to be the same in the config. This was why I was getting errors on the processing screen.

In retrospect, `steps.js` should contain an array of step objects with a `name` property, instead of a map of step names to step objects, to prevent this issue.